cvat/apps/engine/tests/utils.py

```diff
@@ -497,7 +497,8 @@ def check_optional_fields(self: TestCase,
                         ) -> None | NoReturn:
     for k in optional_values.keys():
         # if obj.get(k) and not optional_values[k] == obj[k]:
-        self.assertEqual(obj.get(k, optional_values[k]), optional_values[k]) # coalesce
+        self.assertEqual(obj.get(k, optional_values[k]), optional_values[k],
+                         f"key '{k}' has incorrect expected optional value {obj.get(k)} instead of {optional_values[k]}") # coalesce
```

Comment:
- Please remove commented code.
- It's not totally clear what this function is supposed to check now. Optional values have to be present always, because they have default values, but the values doesn't have to be the defaults.